.cirrus.yml

```diff
@@ -94,17 +94,17 @@ task:
     FILE_ENV: "./ci/test/00_setup_env_native_tidy.sh"
 
 task:
-  name: "Win64 native [msvc]"
+  name: "Win64 native [vs2022]"
   << : *FILTER_TEMPLATE
   windows_container:
-    cpu: 4
+    cpu: 6
```

Comment:
any reason to change this? With 6 cpu it finished 1 minute earlier compared to master ( https://cirrus-ci.com/task/5018066990923776 ), so it seems like a random fluke.

Also, if someone pushes several pull requests at the same time, the scheduler will hit the total cpu limit earlier.

Reply:
Also, looking at the graph, the CPU usage density is less, indicating that the 5th and 6th cpu may be idle most of the time

Reply:
> any reason to change this?

On master, the time required to get the task done with no caches is very close to 2 hours limit.

> With 6 cpu it finished 1 minute earlier compared to master ( https://cirrus-ci.com/task/5018066990923776 ), so it seems like a random fluke.

Such comparison is not correct, because `vcpkg_binary` cache was invalidated for this PR. Functional tests are more then 10 minutes faster now.

> Also, looking at the graph, the CPU usage density is less, indicating that the 5th and 6th cpu may be idle most of the time

Will 5 cpus be OK?

Reply:
> On master, the time required to get the task done with no caches is very close to 2 hours limit.

Ok, no objection then.

Is there a reason why vs22 needs more resources?

Reply:
> Is there a reason why vs22 needs more resources?

On master, with [cleared caches](https://cirrus-ci.com/task/5306029448626176):
- time: 01:51:03
- peak memory usage: 5.10 GB

![Screenshot from 2022-06-26 12-27-08](https://user-images.githubusercontent.com/32963518/175809957-f7966d36-1f48-4c4d-a919-f4b32679038b.png)

With 156bc89788034c509051e0635ed40694b8d717c6 commit, with [cleared caches](https://cirrus-ci.com/task/6092886604251136):
- _\<TBD>_

Reply:
Actually, there is no significant difference in resources at all.

Reply:
I think it might be best to revert this. Looks like now this won't be scheduled until 4 hours later. If the CPU was reduced, so that it was scheduled earlier, it would take 15 minutes longer to run, but still finish earlier.

Reply:
> I think it might be best to revert this. Looks like now this won't be scheduled until 4 hours later. If the CPU was reduced, so that it was scheduled earlier, it would take 15 minutes longer to run, but still finish earlier.

Ok, But a similar situation with ARM task been observed right now.

Reply:
> I think it might be best to revert this.

#25483